### PR TITLE
The Resurrection

### PR DESCRIPTION
**Pull request by marbar3778**
- This PR is meant to resturcture the Spec repo
- Inital idea is to use it to hold published & WIP Papers, hold the spec of tendermint the protocol, and house on going research that may not be published in a paper.

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 # Tendermint Spec
 
+<<<<<<< HEAD
```
Comment (xla) [inline README.md]: These look like merge leftovers.

File: rfc/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Request for Comments (RFC)
+
+RFC stands for `Request for Comments. It is a social device use to float and polish an idea prior to the inclusion into an existing or new spec/paper/research topic.
```
Comment (ValarDragon) [inline rfc/README.md]: I think this is missing a backtick

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Tendermint Spec
 
-This repository contains the paper describing the Tendermint consensus
-algorithm, including formal proofs of its safety and liveness.
+This repository contains specs of the current and future implementation of Tendermint the protocol, papers that have been published by the ICF or AIB and ongoing research in a variety of areas.
 
-For the pdf, see the [latest
-release](https://github.com/tendermint/spec/releases).
+The paper describing the Tendermint consensus algorithm, including formal proofs of its safety and liveness.
```
Comment (milosevic) [inline README.md]: Maybe this sentence can be removed or moved to paper folder.

File: research/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+This folder holds ongoing research and topics of interest. The [Interchain Foundation](https://interchain.io) has research topics that they are willing fund the research for, to find out more you can go to there [funding repo](https://github.com/interchainio/funding/blob/master/research.md). You can apply for a research grant [here](https://docs.google.com/forms/d/e/1FAIpQLSclH4R5G7WgNpKPvXxPPFRA7rAoyX8nNvsJAQJpZNZwWWjFmA/viewform)
+
+## Areas of Interest
+
+Below you will find some areas of interest which we have.
+
+### Tendermint Consensus
+
+- Formal verification of Tendermint consensus [[complete]](https://arxiv.org/abs/1807.04938)
```
Comment (milosevic) [inline research/README.md]: It's more correct to say: Specification of Tendermint consensus algorithm.

File: research/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- BFT Time [[in progress]](https://github.com/tendermint/tendermint/blob/master/docs/spec/consensus/bft-time.md)
```
Comment (milosevic) [inline research/README.md]: Not sure that this list reflects in the best way short/midterm topics of interest. Maybe it's safer to remove it for this PR and we can maybe update it in a separate PR.

File: research/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# Research
+
+This folder holds ongoing research and topics of interest. The [Interchain Foundation](https://interchain.io) has research topics that they are willing fund the research for, to find out more you can go to there [funding repo](https://github.com/interchainio/funding/blob/master/research.md). You can apply for a research grant [here](https://docs.google.com/forms/d/e/1FAIpQLSclH4R5G7WgNpKPvXxPPFRA7rAoyX8nNvsJAQJpZNZwWWjFmA/viewform)
+
+## Areas of Interest
+
+Below you will find some areas of interest which we have.
```
Comment (milosevic) [inline research/README.md]: I would suggest removing the text below as I am not sure that this list reflects in the best way short/midterm topics of interest. We haven't updated it for quite some time, but maybe it's safer to remove it for this PR and we can maybe update it in a separate PR.

File: README.md (hunk 3 above)
Comment (marbar3778) [reply README.md]: will remove this and once move the pdf version into papers/ in a following PR

**Pull request closed without merging**